packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -158,15 +158,21 @@ Registrations are allowed by default, so setting a value of `ALLOW` for the `act
 
 ### error
 
-See [error](/docs/concepts/inline-hooks/#error) for general information on the structure to use for the `error` object.
+See [error](/docs/concepts/inline-hooks/#error) for general information about the structure to use for the `error` object.
 
-For the Registration Inline Hook, the `error` object provides a way of displaying an error message to the end user who is trying to register. If you're using the Okta Sign-In Widget for Profile Enrollment, and have not customized its error handling behavior, only the `errorSummary` of the first `errorCauses` object that your external service returns appears to the end user.
+> **Note:** If you include an error object in your response, no commands are executed and the registration fails. This holds true even if the top-level `errorSummary` and the `errorCauses` objects are omitted.
 
-If you don't return any value for that `errorCauses` object and deny the user's registration attempt through the `commands` object in your response to Okta, the following generic message appears to the end user: "Registration cannot be completed at this time".
```

Comment:
This bullet point has been removed, but is still valid for Classic orgs (from the last time I tested, which was a while ago). I know OIE is the default now, though ... not sure how we handle this. Up to now, we've had both OIE and Classic content bundled together. However, if decision is to remove Classic behaviour, that's fine.

Reply:
it's still there.